src/app/components/collapsible.js

```diff
@@ -9,7 +9,7 @@ import React, { useState } from "react";
  * @param {React.ReactNode} prop.children  - The content to display of the container.
  * @returns {JSX.Element} The Collapsible component.
  */
-export function Collapsible({ title, children }) {
+export function Collapsible({ title, showName = 'Show', hideName = 'Hide', children }) {
```

Comment:
Can you please move these optional params to the end of the function signature, then please update the jsdoc above it with descriptions of the new parameters?

Reply:
Oh, and please double-quote the strings while you are at it

Reply:
Still learning JS and/or NextJS.. sounds like `children` IS a valid variable that is assumed here.. Not for me to be confused by "also include everything else that might came along when someone uses this function, aka `...prop`).